frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/RediscoverSuggester.java

```diff
@@ -62,13 +71,49 @@ private void queueSuggestion(Path macheteFilePath) {
       queueDiscoverOperation.run();
     } else { // closing dialog goes here too
       LOG.info("Rediscover declined from dialog");
-      macheteFilePath.setFileModificationDate(System.currentTimeMillis());
+      setMacheteFileModificationDate(macheteFilePath, System.currentTimeMillis());
     }
   }
 
+  @UIEffect
+  private void setMacheteFileModificationDate(Path macheteFilePath, long timeMillis) {
```

Comment:
Doesn't need to be named `...Machete...`, this method doesn't need to know that the file being updated is machete file; also, the method can be static, plus the `timeMillis` param is always `System.currentTimeMillis()`